src/Formatters/ICodeFormatter.cs

```diff
@@ -14,7 +15,7 @@ internal interface ICodeFormatter
         /// <summary>
         /// Applies formatting and returns a formatted <see cref="Solution"/>.
         /// </summary>
-        Task<Solution> FormatAsync(
+        Task<(Solution Solution, List<FormattedFile> FormattedFiles)> FormatAsync(
```

Comment:
Another instance of style choice. I saw some precedent for using tuples, so decided to use it here. If you would prefer that I created a new class for this, let me know.